docs/guide/glossary.md

```diff
@@ -36,4 +36,4 @@ The actionbar shows up only on the frontend of sites. It is there to provide con
 
 In development environments, when you're developing Calypso, you'll see a badge in the bottom right corner.
 
-Previous: [The Technology Behind Calypso](tech-behind-calypso.md) Next: [Contributing to Calypso](../../.github/CONTRIBUTING.md)
+Previous: [Design Patterns](design-patterns.md) Next: [Contributing to Calypso](../../.github/CONTRIBUTING.md)
```

Comment:
Oh, didn't know these had to be manually updated :(

Reply:
High tech!